scripts/BusFruCallouts.pm

```diff
@@ -238,29 +238,24 @@ sub setupBusWithParentTarget
 
                     if ($parent eq '')
                     {
-                        # Look for the exception of an OCMB on the planar
-                        my $die = "TRUE";
-                        my $conn_target = $conn->{$busPath};
+                        # Look for the exception case of a valid OCMB target being on the planar.
```

Comment:
I hope we are following [this](https://github.com/open-power/pdata/pull/27#discussion_r1209675362) comment to check the hw-isolation impacts.

Reply:
We have a story to handle the hw isolation changes.